### DISCUSSION TITLE
Request Transfer of Cake.EntityFrameworkCore into cake-contrib org

### DISCUSSION THREAD

**Discussion by louisfischer** [cake-build/cake#3262] [Extension requests]
I would like to request to transfer my new Addin for Cake into the Cake Contribution Organization. 
Would someone be able to assist with this?

https://github.com/louisfischer/Cake.EntityFrameworkCore

**Comment by augustoproiete** [comment]
@louisfischer If you transfer to my user `augustoproiete` I can transfer to the Cake Contrib organization

**Reply by louisfischer** [reply]
@augustoproiete done

**Reply by augustoproiete** [reply]
Perfect, thanks. I transferred the  repo to the Cake-Contrib org https://github.com/cake-contrib/Cake.EntityFrameworkCore and you should have full admin access to it
